test/parallel/test-https-agent-unref-socket.js

```diff
@@ -0,0 +1,13 @@
+'use strict';
+
+const common = require('../common');
+if (!common.hasCrypto)
+  common.skip('missing crypto');
+
+const https = require('https');
+
+const request = https.get('https://www.google.com');
```

Comment:
```suggestion
const request = https.get('https://example.com');
```

Could we use `https://example.com` for test?  Since`https://www.google.com` is unaccessible in my area without proxy, it makes testing hanging. `example.com` is reserved by IANA, we could assume that it is always avaliable.

Reply:
@Ayase-252 Done. :)